Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,205 @@
+              app:layout_constraintTop_toBottomOf="@+id/subject_text_view"
+              tools:text="3 more messages" />
+          <!-- TODO: Set up Option Menu functionality -->
+          <TextView
+              android:id="@+id/subject_text_view"
+              android:layout_width="0dp"
+              android:layout_height="wrap_content"
+              android:gravity="center"
+              android:text="@{email.subject}"
+              android:textAppearance="@style/TextAppearance.Reply.EmailSubject"
+              app:layout_constraintEnd_toEndOf="parent"
+              app:layout_constraintStart_toStartOf="parent"
+              app:layout_constraintTop_toTopOf="parent"
+              tools:text="High school reunion" />
+          <Button
+              android:id="@+id/navigation_icon"
+              style="@style/Widget.App.Button.OutlinedButton.IconOnly"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              app:icon="@drawable/ic_arrow_back"
+              app:iconSize="24dp"
+              app:layout_constraintBottom_toBottomOf="parent"
+              app:layout_constraintStart_toStartOf="parent"
+              app:layout_constraintTop_toTopOf="parent" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+      </androidx.appcompat.widget.Toolbar>
+    </com.google.android.material.appbar.AppBarLayout>
+    <!--  TODO: Add RecyclerView logic here  -->
+    <com.google.android.material.card.MaterialCardView
+        style="?attr/materialCardViewElevatedStyle"
+        android:id="@+id/email_card_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/grid_2"
+        android:transitionName="@string/email_card_detail_transition_name"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/email_app_bar_layout">
+      <androidx.constraintlayout.widget.ConstraintLayout
+          android:id="@+id/email_content_constraint_layout"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:paddingTop="@dimen/grid_3"
+          android:paddingBottom="@dimen/grid_3"
+          android:paddingHorizontal="@dimen/grid_2"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@+id/email_app_bar_layout">
+        <ImageView
+            android:id="@+id/sender_profile_image_view"
+            android:layout_width="@dimen/email_sender_profile_image_size"
+            android:layout_height="@dimen/email_sender_profile_image_size"
+            android:contentDescription="@string/email_sender_profile_content_desc"
+            android:scaleType="centerCrop"
+            app:glideSrc="@{email.sender.avatar}"
```

Comment:
This ImageView does need the `app:glideCircularCrop="@{true}" to make it round. Apologies if I incorrectly suggested you remove it previously!